equinor/design-system
# ♿️  Remove fixed height on Button and Table

My first thought was to add this to `@equinor/eds-labs` – but we already have a button component there with the CSS spike. We could potentially rename that to something else… or publish this to `@equinor/eds-core-react` to a `next` release so they can test it in Envis 🤔

How to test:
1. Spin up a Codespace from this pr
1. Wait for the dust to settle 
1. Run `pnpm storybook:core-react` in the terminal and open the preview in a browser 
1. Change the root font size in the browser* and observe how the button size changes

_*In Chrome, go to Settings → Appearance → Font size_

Resolves #1858
Resolves #1867
